src/main.rs

```diff
@@ -316,10 +323,12 @@ async fn exec() -> Result<(), Error> {
     };
 
     // connect to rusk
-    let rusk = if cfg.rusk.ipc_method == "uds" {
-        rusk_uds(&cfg.rusk.rusk_addr).await
-    } else {
-        rusk_tcp(&cfg.rusk.rusk_addr, &cfg.rusk.prover_addr).await
+    let ipc = cfg.rusk.ipc_method.as_str();
+    let rusk = match ipc {
+        #[cfg(not(windows))]
+        "uds" => rusk_uds(&cfg.rusk.rusk_addr).await,
+        "tcp_ip" => rusk_tcp(&cfg.rusk.rusk_addr, &cfg.rusk.prover_addr).await,
+        _ => panic!("IPC method \"{}\" not supported", ipc),
```

Comment:
I'm not sure about this.
IIUC the PR description this code should use TCP/IP on Windows even if `uds` is specified.

To me seems that this code will panic if the conditions above mentioned are met